pipeline/dags/probe_api.py

```diff
@@ -0,0 +1,126 @@
+import pendulum
+
+from airflow.decorators import dag, task
+from airflow.operators import empty
+
+from dag_utils.virtualenvs import PYTHON_BIN_PATH
+
+default_args = {}
+
+
+@task.external_python(
+    python=str(PYTHON_BIN_PATH),
+)
+def store_probe_results(get_requests: list[str], today: str) -> None:
+    import json
+
+    import furl
+    import requests
+
+    from airflow.models import Variable
+
+    from dag_utils import s3
+
+    BASE_URI = "https://api.data.inclusion.gouv.fr"
+    token_probe = Variable.get("DATA_INCLUSION_API_PROBE_TOKEN")
+    headers = {
+        "Authorization": f"Bearer {token_probe}",
+    }
+    for get_request in get_requests:
+        url = furl.furl(f"{BASE_URI}/{get_request}")
+
+        response = requests.get(url.add({"size": 100}).url, headers=headers)
```

Comment:
supernit mais j'ai la manie de documenter les choix de variable numérique ("comments tell you why") avec un petit commentaire dans le code, juste pour pas que on se creuse la tronche à comprendre si on peut augmenter ou pas cette valeur un jour et à chercher partout dans notion, sur slack, etc... ^^